src/main/pages/user-management/permissions.adoc

```diff
@@ -11,37 +11,35 @@ folder: user-management
 [id="overview"]
 == Overview
 
-Permissions are used to control user actions. Rather than providing a fixed set of roles we use a broader set of permissions that can be applied in any combination of objects to establish the security model you need.
+Permissions are used to control the actions of users and establish a security model. You can define _who_ is allowed to do _what_. You can control resources managed by Che and allow certain actions by assigning permissions to users.
 
-Che also provides a mechanisms and layers which allow to define "who" is allowed to do "what". Any user and administrator can control resources managed by Che and allow certain actions or behaviors for other users or groups. For example as workspace owner you can grant other users permission to see and/or use your workspace.
-
-Permissions can be applied to:
+Permissions can be applied to the following:
 
 * Workspace
 * Organization
 * Stack
 * System
 
 [id="workspace-permissions"]
-== Workspace Permissions
+== Workspace permissions
 
-The user who creates a workspace is the _workspace owner_ and has all permissions by default. Workspace owners can invite other users into the workspace and control their permissions for the workspace.
+The user who creates a workspace is the workspace owner. The workspace owner has the following permissions by default:  read, use, run, configure, setPermissions, and delete. Workspace owners invite users into the workspace and control workspace permissions for each user.
```

Comment:
Should the permissions be stylized in some way?

Reply:
I checked the IBM style guide for reference